docs/source/usrman/mpi4py.futures.rst

```diff
@@ -421,18 +421,39 @@ scanline is dumped to disk.
                for line in image:
                    f.write(line)
 
-The recommended way to execute the script is using the :program:`mpiexec`
+The recommended way to execute the script is by using the :program:`mpiexec`
 command specifying one MPI process and (optional but recommended) the desired
-MPI universe size [#]_. ::
+`MPI universe size <https://www.mcs.anl.gov/research/projects/mpi/mpi-standard/mpi-report-2.0/node111.htm>`_,
+which is provided either by a batch system or is set by the user via
+command-line arguments or environment variables. Below we provide
+examples for MPICH and Open MPI as well as `mpi4py`-specific way [#]_.
```

Comment:
And here he have potential confusion. The mpi4py-specific way is unrelated to the MPI universe size concept. This wording does not make it clear. Please remove  `as well as mpi4py-specific way`. Discussing MPI4PY_MAX_WORKERS at the end will still be good, but you have to do a better job of not confusing it with the MPI universe size.

Reply:
I've removed the phrase `as well as mpi4py-specific way`.